Essentials/src/com/earth2me/essentials/commands/Commandhat.java

```diff
@@ -35,6 +35,10 @@ protected void run(final Server server, final User user, final String commandLab
         } else {
             final ItemStack hand = user.getItemInHand();
             if (hand != null && hand.getType() != Material.AIR) {
+                if (user.isAuthorized("essentials.hat.exempt." + hand.getType().name().toLowerCase())) {
```

Comment:
I had previously approved this, but "hat exempt" doesn't make a lot of sense. Could change it to `essentials.hat.prevent-type.<type>`? In addition, this check needs to be replicated in `EssentialsPlayerListener` where the hat handling happens.